%YAML:1.0

#common parameters
savePath: "/home/hyj/all_ros_ws/LaserCamCal_ws/src/log/"
bag_path: "/home/hyj/all_ros_ws/LaserCamCal_ws/src/2019-07-12-11-36-17.bag"

scan_topic_name: "/scan"
img_topic_name: "/mynteye/left/image_raw"

# tag info
tag_type: 2 # 1: kalibr_tag, 2: apriltag
tag_size: 0.165 # tag size, unit meter
tag_spacing: 0.3 # tag spacing, only used in kalibr_tag. For details, please see kalibr tag description.
black_border: 1 # if you use kalibr_tag black_boarder = 2; if you use apriltag black_boarder = 1

#camera calibration 
model_type: PINHOLE
camera_name: camera
image_width: 752
image_height: 480

distortion_parameters:
   k1: 0.0
   k2: 0.0
   p1: 0.0
   p2: 0.0

projection_parameters:
   fx: 367.049931000148
   fy: 366.94446918887405
   cx: 368.7202381120387
   cy: 241.13814795878562

